### PR TITLE
Featured branch

### PR DESCRIPTION
**Pull request by Mike-devloper**
Add linters configurations
Add index.html file
Add styles.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <div>
+            <h1 id="welcome">Welcome</h1>
+        </div>
+        <img src="icons/menu.png" class="material-icons" alt="menu icon">
+    </header>
+    <section id="headline">
+        <div class="center">
+                <h2 id="name"><b> Hey there. I’m Micheal</b> <br>
+                    <b id="job">I’m a software developer </b>
+                </h2>
```
Comment (levy002) [inline index.html:24]: - [ ] Kindly change this `h2` to `h1` as this is the main heading on the page it should be wrapped into `h1` tag. For more information about this heading please check [here](https://www.reliablesoft.net/h1-tag/).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        </div>
+        <div class="center"><h3 id="story"> I can help you build a product , feature or website Look through some of my work and
+            experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</h3>
+        </div>
```
Comment (levy002) [inline index.html:29]: - [ ] Kindly wrap this paragraph into `p` as it is a paragraph it should be wrapped into `p` tag. This way will be semantically correct and will increase website accessibility for screen readers and other technologies.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <div id="icons">
+            <i class="fa fa-twitter"></i>
+            <i class="fa fa-linkedin"></i>
+            <i class="fa fa-medium"></i>
+            <i class="fa fa-github"></i>
+            <i class="fa fa-angellist"></i>
+        </div>
```
Comment (levy002) [inline index.html:36]: - Kindly group all your social media links into `ul` and `li` tags and remember that each should also be a link so it means it should be wrapped into an `a` tag.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Mike-devloper   , :wave:
### I'm Levy, your PR reviewer

Awesome :sunglasses: Good job so far. 
There are some issues that you still need to work on to go to the next project but you are almost there!

You can do it

## Highlights
- Good commit messages :heavy_check_mark: 
-  Linters are green :heavy_check_mark: 
- Nice design :heavy_check_mark: 
- Correct usage of Github flow :heavy_check_mark: 
## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                    <a> <i class="fa fa-medium"></i></a>
+                </li>
+                <li> 
+                    <a><i class="fa fa-github"></i></a>
+                </li>
+                <li> 
+                    <a> <i class="fa fa-angellist"></i></a>
+                </li>
+            </ul>
+
```
Comment (zieeco) [inline index.html:50]: - Please, note the anchor is missing the href attribute as these are links to your social media platform. kindly add the corresponding href to each of the icons. if you don't have some, you could make it a dead link but would be update later in the future

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Mike-devloper,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+                <li> 
+                    <a href="www.medium.com/mike-dev"> <i class="fa fa-medium"></i></a>
+                </li>
+                <li> 
+                    <a href="www.github.com/mike-dev"><i class="fa fa-github"></i></a>
+                </li>
+                <li> 
+                    <a href="www.angellist.com/mike-dev"> <i class="fa fa-angellist"></i></a>
+                </li>
+            </ul>
```
Comment (Bondok6) [inline index.html:49]: - [OPTIONAL] Great job adding the `href` attribute here 💯. However, According to the requirements you must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468), So Kindly, update the style of `icons` id. I suggest adding `list-style: none` to remove the dots and change the icons colors to match your figma design 👍👌

```
#icons{
list-style: none
}
```

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Mike-devloper  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the requirements by the previous reviewers ✅
- No Linter errors ✅
- Your repo is professional ✅
- Well structured files ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (oguntayomicheal) [reply index.html]: Hello @levy002, thanks for the review. I will make sure to implement all the suggested changes.

File: index.html (hunk 4 above)
Comment (oguntayomicheal) [reply index.html]: Hi @zieeco, thanks for drawing my attention to that. I will add the href attributes to the `anchor  tags`

File: index.html (hunk 5 above)
Comment (oguntayomicheal) [reply index.html:49]: Hello @Bondok6, thank you for approving my project. I will surely implement the suggested changes.

**Pull request merged**